lib/tools/adb-commands.js

```diff
@@ -805,10 +805,23 @@ methods.setHttpProxy = async function (proxyHost, proxyPort) {
   await this.setSetting('system', 'global_http_proxy_port', proxyPort);
 };
 
+/*
+* Set device property.
+*   namespace: one of {system, secure, global}, case-insensitive
+*   setting: property name
+*   value: property value
+* https://developer.android.com/reference/android/provider/Settings.html
+*/
 methods.setSetting = async function (namespace, setting, value) {
-  await this.shell(['settings', 'put', namespace, setting, value]);
+  return await this.shell(['settings', 'put', namespace, setting, value]);
 };
 
+/*
```

Comment:
Please rebase your branch with the recent master to see docstring examples. We do follow [JSDoc tutorial](http://usejsdoc.org/) while creating docstrings.

Reply:
oops...